### DISCUSSION TITLE
Branches no GitHub: Melhores Práticas e Estratégias para Desenvolvimento Colaborativo

### DISCUSSION THREAD

**Discussion by Alxdelira** [community/community#120286] [Projects and Issues] [answered]
### Select Topic Area

Product Feedback

### Body

As branches são uma parte fundamental do fluxo de trabalho no GitHub, permitindo que equipes colaborem de forma eficiente em projetos de código aberto. Nesta discussão, gostaria de explorar diferentes estratégias e melhores práticas para o uso de branches no GitHub. Você prefere seguir um modelo de GitFlow com branches de feature, develop e release, ou opta por uma abordagem mais simplificada? Como você gerencia conflitos entre branches e mantém um histórico limpo de commits? Compartilhe suas experiências, desafios e dicas para aproveitar ao máximo as branches no GitHub!

Qual é o seu fluxo de trabalho preferido ao trabalhar com branches no GitHub?
Como você decide quando criar uma nova branch e quando mesclá-la de volta ao branch principal?
Quais são os desafios comuns que você enfrenta ao lidar com conflitos entre branches?
Você tem alguma estratégia específica para manter um histórico limpo de commits em suas branches?
Você já experimentou diferentes modelos de branch, como GitFlow, e qual foi sua experiência com eles?
Como você comunica e coordena o trabalho em branches com membros da equipe ou colaboradores externos?

**Comment by ettaboyle** [comment, marked as answer]
Hi @Alxdelira, thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
